### ISSUE TITLE
LEGO City Undercover (578330)

### ISSUE THREAD

**Issue by leillo1975** [ValveSoftware/Proton#1961]
# Whitelist Request
- Name of the game to be whitelisted: LEGO City Undercover
- Steam AppID of the game: 578330

## System Information
- GPU: NVIDIA GTX 1050Ti
- Driver/LLVM version: nvidia 396.54
- Kernel version: 4.18
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/leillo1975/bd9c738c5d2aae6b44ef0569c2c218c7
- Proton version:3.16-4 Beta

## I confirm:
- [x] that pressing the `Play` button in the Steam client is sufficient.
- [ ] that [runtime config options](https://github.com/ValveSoftware/Proton#runtime-config-options) are necessary to run the game.
- [x] that no workarounds other than the mentioned ones are necessary.

## Issues
- [x] I haven't experienced any issues.
- [x] There are no issues left open for this game.
- [ ] Although I consider the gaming experience equal to Windows there are remaining issues:

**Comment by leillo1975** [comment]
Works perfect. No issues, no fixes, no workarounds

**Comment by linuxgaming** [comment]
What distro, if you don't mind me asking? For me, it doesn't launch on SteamOS (steamos beta + steam beta) with Proton 3.16-4 but works well with 3.7 (albeit the background sound is always on max volume with no means to change it). I switched to 3.16 in hope it would fix the sound issue but instead it stopped working altogether.

**Comment by kisak-valve** [comment]
From the linked [System Information](https://gist.github.com/leillo1975/bd9c738c5d2aae6b44ef0569c2c218c7): Ubuntu 18.04.1 LTS (64 bits)

**Comment by ghost** [comment]
@kisak-valve 

# Whitelist Request
- LEGO City Undercover
- 578330

## System Information
- GPU: RX 570
- Driver/LLVM version: Mesa 18.3 & LLVM 7
- Distro: Sabayon
- Link to full system information report as [Gist](https://gist.github.com/Alexander88207/72c104e4d0f59898cc4ede2fce2de25a)
- Proton version: 3.16-5

## I confirm:
- [✔] that pressing the `Play` button in the Steam client is sufficient.

## Issues
- [✔] I haven't experienced any issues.
- [✔] There are no issues left open for this game.

All audio issues are now fixed! :+1:

**Comment by linuxgaming** [comment]
I can confirm that with Proton 3.16-5 the game runs perfect on SteamOS and all sound issues are gone. :-)

## System Information
- GPU: GTX 970
- Driver/LLVM version: 396.54
- Distro version: SteamOS 2.166
- Proton version: 3.16-5

**Comment by ghost** [comment]
̶f̶̶r̶̶o̶̶m̶̶ ̶̶a̶̶ ̶̶c̶̶e̶̶r̶̶t̶̶a̶̶i̶̶n̶̶ ̶̶m̶̶i̶̶s̶̶s̶̶i̶̶o̶̶n̶̶,̶̶ ̶̶i̶̶t̶̶ ̶̶a̶̶l̶̶w̶̶a̶̶y̶̶s̶̶ ̶̶c̶̶r̶̶a̶̶s̶̶h̶̶e̶̶s̶̶,̶̶ ̶̶p̶̶r̶̶e̶̶v̶̶e̶̶n̶̶t̶̶i̶̶n̶̶g̶̶ ̶̶m̶̶e̶̶ ̶̶t̶̶o̶̶ ̶̶c̶̶o̶̶n̶̶t̶̶i̶̶n̶̶u̶̶i̶̶n̶̶g̶̶ ̶̶t̶̶h̶̶e̶̶ ̶̶g̶̶a̶̶m̶̶e̶̶.̶

̶e̶̶d̶̶i̶̶t̶̶:̶̶ ̶̶i̶̶s̶̶ ̶̶r̶̶e̶̶l̶̶e̶̶a̶̶t̶̶e̶̶d̶̶ ̶̶t̶̶o̶̶ ̶̶d̶̶x̶̶v̶̶k̶̶ ̶̶t̶̶u̶̶r̶̶n̶̶ ̶̶i̶̶t̶̶ ̶̶o̶̶f̶̶f̶̶ ̶̶f̶̶o̶̶r̶̶ ̶̶t̶̶h̶̶i̶̶s̶̶ ̶̶o̶̶n̶̶e̶̶ ̶̶m̶̶i̶̶s̶̶s̶̶i̶̶o̶̶n̶̶ ̶̶w̶̶i̶̶t̶̶h̶̶ ̶̶p̶̶r̶̶o̶̶t̶̶o̶̶n̶̶_̶̶u̶̶s̶̶e̶̶_̶̶w̶̶i̶̶n̶̶e̶̶d̶̶3̶̶d̶=̶1̶̶ ̶%̶c̶̶o̶̶m̶̶m̶̶a̶̶n̶̶d̶%

**Comment by linuxgaming** [comment]
> from a certain mission, it always crashes, preventing me to continuing the game.

We 100%'ed the game on SteamOS so it certainly is possible. Try different proton flags as well as different resolutions (this one might require manually editing the game's config), the crashes occur on Windows too, so it might not even be fixable in Proton. There are lots of topics mentioning possible solutions: https://steamcommunity.com/app/578330/discussions/search/?q=crash&gidforum=135512931358081225&include_deleted=1

**Comment by ghost** [comment]
> > from a certain mission, it always crashes, preventing me to continuing the game.
> 
> We 100%'ed the game on SteamOS so it certainly is possible. Try different proton flags as well as different resolutions (this one might require manually editing the game's config), the crashes occur on Windows too, so it might not even be fixable in Proton. There are lots of topics mentioning possible solutions: https://steamcommunity.com/app/578330/discussions/search/?q=crash&gidforum=135512931358081225&include_deleted=1

Okkay after a new installation of my distro everything works again :D

**Comment by nariox** [comment]
I wanted to add that the game works with ESYNC, but crashes on cutscenes with FSYNC on my machine.

DXVK and D3D both work, but DXVK is much faster.

**Comment by kisak-valve** [comment]
Hello @nariox, can you also add a note to #2922 with your findings.

**Comment by nariox** [comment]
Yes, @kisak-valve. I have added the info on the issue tracker yesterday, but didn't have time to reply here. Let me know if any other information is desired (traces, debug log, etc).

**Comment by sphaero** [comment]
With a default install of a steamdeck I've experienced that game progress isn't saved. We need to restart from the beginning every time.

It's fixed now... update I guess?

**Comment by rinsuki** [comment]
## System Information
- GPU: AMD AMD Radeon 780M Graphics (integrated)
- Video driver version: Mesa 25.1.4-arch1.1
- Kernel version: 6.15.3-arch1-1
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/rinsuki/2d7216bac764b5417c594dcf73dcab6d

---

The OP movie is glitched in Proton 9.0-4, Proton 10-1 (beta), and Proton Hotfix, but works fine in Proton 8.0-5 and/or Experimental.

**Comment by kisak-valve** [comment]
Hello @rinsuki, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the regression, and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.) Also, can you grab a screenshot of the misrendering or elaborate on the nature of the glitch you're seeing?

**Comment by rinsuki** [comment]
Sorry, I forgot; heres is the log: [steam-578330.log](https://github.com/user-attachments/files/20963094/steam-578330.log)

Screenshot (taken by Steam's F12 key)

![Image](https://github.com/user-attachments/assets/b67ab232-cd61-4199-b6cb-a623c7c7c266)

**Comment by alasky17** [comment]
@rinsuki Thank you for the regression report! It turns out this is already fixed in upstream dxvk which is why it was working in experimental.  It is still great for us to know when things are regressed on a stable release - thank you!

**Comment by kisak-valve** [comment]
LEGO City Undercover uses single controller input for both players

Issue transferred from https://github.com/ValveSoftware/Proton/issues/9387.
@TitanNano posted on 2026-01-10T21:01:43:

# Compatibility Report
- Name of the game with compatibility issues: LEGO City Undercover
- Steam AppID of the game: 578330

## System Information
- GPU: RTX 5060
- Video driver version: NVIDIA 580.119.02
- Kernel version: 6.17.12
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/TitanNano/2f5506b1dc6aaf7b964d78b7c565ab98
- Proton version: 10.0-3

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-578330_lego_city.log](https://github.com/user-attachments/files/24546420/steam-578330_lego_city.log)

## Symptoms
When launched through Steam Big Picture Mode, the game recognized the single connected Steam controller as input for both players.

- This issue did not exist before Proton 10. (i.e., Proton 9 works fine)
- This issue does not occur when launching the game through the regular desktop mode of Steam.

## Reproduction
1. start big picture mode
2. Start LEGO City Undercover
3. Load a save game or start a new one.
4. In the game world, press any of the controller buttons. (Moving the analog stick does not trigger the issue.)
5. The second player gets enabled.
6. Both players are controlled by the same controller.

**Comment by simifor** [comment]
@TitanNano what controller are you using and how are you connecting it to your computer? (bluetooth/usb)

**Comment by TitanNano** [comment]
@simifor I'm using one of the original Steam controllers. 

EDIT: The controller is wirelessly connected via the USB dongle.
